cmd/crank/beta/trace/internal/printer/default_test.go

```diff
@@ -52,15 +52,15 @@ func TestDefaultPrinter(t *testing.T) {
 			want: want{
 				// Note: Use spaces instead of tabs for intendation
 				output: `
-NAME                                                   READY   SYNCED   STATUS                                                
-ObjectStorage/test-resource (default)                  True    True                                                           
-└─ XObjectStorage/test-resource-hash                   True    True                                                           
-   ├─ Bucket/test-resource-bucket-hash                 True    True                                                           
-   │  ├─ User/test-resource-child-1-bucket-hash        False   True     SomethingWrongHappened: Error with bucket child 1     
-   │  ├─ User/test-resource-child-mid-bucket-hash      True    True     AllGood                                               
-   │  └─ User/test-resource-child-2-bucket-hash        False   True     SomethingWrongHappened: Error with bucket child 2     
-   │     └─ User/test-resource-child-2-1-bucket-hash   False   True     SomethingWrongHappened: Error with bucket child 2-1   
-   └─ User/test-resource-user-hash                     True    True                                                           
+NAME                                                   READY   SYNCED   STATUS                                              
+ObjectStorage/test-resource (default)                  True    True                                                         
+└─ XObjectStorage/test-resource-hash                   True    True                                                         
+   ├─ Bucket/test-resource-bucket-hash                 True    True                                                         
+   │  ├─ User/test-resource-child-1-bucket-hash        False   True     SomethingWrongHappened: Error with bucket child 1   
+   │  ├─ User/test-resource-child-mid-bucket-hash      True    False    CantSync: Sync error with bucket child mid          
+   │  └─ User/test-resource-child-2-bucket-hash        False   True     SomethingWrongHappened: Error with bucket child 2   
+   │     └─ User/test-resource-child-2-1-bucket-hash   -       True                                                         
```

Comment:
nit: Should we still have a test for the real `Unknown`s :)

Reply:
done: https://github.com/crossplane/crossplane/pull/4849/commits/f8998ceee913e53c1c4585e448f5d0936c44fb38